.github/workflows/pypi_install.yml

```diff
@@ -0,0 +1,83 @@
+# Let's upload nestpy to PyPi to make it pip installable
+# Mostly based on https://github.com/marketplace/actions/pypi-publish
+# Some of this is a bit clunky, we have both Linux and MacOS build.
+# We apply the following strategy:
+#  - For Mac, run both MacOS-latest (11.xx) and MacOS-10.15. The latter
+#    seems to be required for to properly build the wheels commonly used.
+#    For each python version and MacOS version, we run a separate action
+#    to complete all the required wheels.
+#  - For linux, use a nice pre-defined docker image from
+#    "RalfG/python-wheels-manylinux-build@v0.4.2-manylinux2014_x86_64"
+#    to build the wheels. This allows us to build several wheels in one
+#    go. We build newer python versions explicitly with numpy 1.21
+#    (though this is probably not strictly required).
+
+name: Pipy
+on:
+  workflow_dispatch:
+  release:
+    types: [ published ]
+  push:
+    tags:
+      - '*'
```

Comment:
@sophiafarrell I'm not sure if you want to have this, it would automatically publish to pipy as soon as you make a new tag. 

If by removing these lines, you would first have to write a new release here https://github.com/NESTCollaboration/nestpy/releases before it gets uploaded.

Reply:
I think this is actually ideal, so that beta versions and such don't need to have full-on publications to upload to pipy.

Reply:
Of course still writing releases will be good to do for documentation purposes, but I don't think it should be as before, where it was the required step for triggering an upload to pipy.